### ISSUE TITLE
Error compiling with Visual Studio 2017 - Impossible to find value 'Ctrl' requested

### ISSUE THREAD

**Issue by bovirus** [HakanL/resxtranslator#41]
src\Windows\MainWindow.resx(596,5): 
error MSB3103: 
file Resx not valid. 
Impossible to find value 'Ctrl' requested. 
Row 596, position 5.

**Comment by Klocman** [comment]
The build works fine for me. Make sure your VS is fully updated and that you have the .NET 4.x targeting packs installed.

**Comment by bovirus** [comment]
Visual Studio 2017 15.9.20 installed and updated.
.Net Framnework installed (4.8 witHW indows 10).
Is requested a specfic version of Framework.net?

It's quite strange because also the original file get the same error.

**Comment by Klocman** [comment]
I'm currently using Windows 10, it shouldn't affect anything. The targeting packs are not the same as the .Net version used by your OS. You need to install those through the Visual Studio installer (Tools > Get tools and features).

**Comment by bovirus** [comment]
I already reinstalled from scratch Visual Studio 2017.

What kind of Framework.net are yoiu using in Windows 10? 4.8?
Because in teh source there are referenecs to 3.5. and 4.0.0.

When you mentioend to install it from "Tools" -> "Get tools and features" and what modules are referring?

**Comment by Klocman** [comment]
The version of .Net in Windows 10 does not matter, you should always have the newest version installed and it will run apps compiled for all previous versions without issues.

You should have `.NET Destop Development` workload and all targeting packs for .NET Framework installed.
![Capture](https://user-images.githubusercontent.com/14913904/75351712-dcc6fd80-58a8-11ea-86d0-48fde2c8e48c.PNG)

**Comment by bovirus** [comment]
I checked that all framework.net are like yours.

In src\CellEditor\CellEditorWindows.resx tehre is a mention about .net framework 3.5.1

<data name="&gt;&gt;textBoxString.Type" xml:space="preserve">
    <value>ScintillaNET.Scintilla, ScintillaNET, Version=3.5.10.0, Culture=neutral, PublicKeyToken=null</value>
  </data>

Is it normal?

Loading the .sln file I got this error


J:\Development\VisualC\resxtranslator-master\src\Properties\Settings.Designer.cs	232	Active
Warning		The primary reference "J:\Development\VisualC\resxtranslator-master\bin\Debug\PortableSettingsProvider.dll" could not be resolved because it was built against the ".NETFramework,Version=v4.6.1" framework. This is a higher version than the currently targeted framework ".NETFramework,Version=v4.0,Profile=Client".	ResxTranslator			
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'PortableSettingsProvider' could not be found (are you missing a using directive or an assembly reference?)	ResxTranslator
